convert_test.go

```diff
@@ -75,9 +75,10 @@ func TestFromTime(base *testing.T) {
 	base.Parallel()
 
 	type testCase struct {
-		Time  string
-		Date  date.Date
-		Error string
+		Time     string
+		Date     date.Date
+		Timezone timezoneMetadata
```

Comment:
I chose to go with a nested struct here so that **MOST** test cases could ignore it (there are 3 fields to set).